### ISSUE TITLE
Add support for wildcards in beet.json

### ISSUE THREAD

**Issue by Kore-Development** [mcbeet/beet#104]
```json
{
    "data_pack": {
        "load": ["src", "vendor/*.zip"]
    }
}
```
With this beet.json file, all the files ending with .zip in the vendor folder should be loaded.

**Comment by vberlier** [comment]
That's a pretty nice suggestion! Shouldn't take that much time to implement either.

**Comment by vberlier** [comment]
This is now implemented in v0.36.0.
